### ISSUE TITLE
[JENKINS-30394] Browser hangs when opening build log with Timestamper plugin installed

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/timestamper-plugin#436]
<p>After installing the plugin and enabling the feature in the job, it seems like we are not able to view the logfile after the build had been completed.</p>

<p>Attached is a screeshot that shows the browser hanging while trying to access the console output "log".</p>

<p>Uninstalling the Plugin fixes the issue.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jyerabutar">jyerabutar</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-30394" target="_blank">Browser hangs when opening build log with Timestamper plugin installed</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/stevengbrown.png" title="stevengbrown's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=stevengbrown">stevengbrown</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Critical
<li><b>component(s)</b>: timestamper-plugin
<li><b>label(s)</b>: plugin
<li><b>resolution</b>: Cannot Reproduce
<li><b>resolved</b>: 2016-01-20T03:48:15+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20251218-075729
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>After installing the plugin and enabling the feature in the job, it seems like we are not able to view the logfile after the build had been completed.</p>

<p>Attached is a screeshot that shows the browser hanging while trying to access the console output "log".</p>

<p>Uninstalling the Plugin fixes the issue.</p></pre>
</details>
</details>
<details><summary><i>environment</i></summary>

```
Jenkins Version 1.626<br/>
Timestamper Plugin Version 1.7.2
```
</details>
<details><summary><i>1 attachment</i></summary>

- [Screenshot 2015-09-10 14.22.39.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/30646/Screenshot%202015-09-10%2014.22.39.png)
  > ![Screenshot 2015-09-10 14.22.39.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-last/refs/heads/main/attachments/30646/Screenshot%202015-09-10%2014.22.39.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/stevengbrown.png" title="stevengbrown's avatar" /> stevengbrown:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-30394?focusedId=236739&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-236739">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<ul class="alternate" type="square">
	<li>Which operating system and browser are you using?</li>
	<li>Does anything appear in the Javascript console?</li>
</ul>

</pre>
</details>

</ul>
</details>

<ul class="alternate" type="square">
	<li>Which operating system and browser are you using?</li>
	<li>Does anything appear in the Javascript console?</li>
</ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>jyerabutar:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-30394?focusedId=236810&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-236810">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<ul class="alternate" type="square">
	<li>Which operating system and browser are you using?
&gt; We have tested this on both mac and windows.</li>
</ul>


<ul class="alternate" type="square">
	<li>Does anything appear in the Javascript console?
&gt; A screenshot has been attached with the output in the JavaScript console.</li>
</ul>


</pre>
</details>

</ul>
</details>

<ul class="alternate" type="square">
	<li>Which operating system and browser are you using?<br/>
> We have tested this on both mac and windows.</li>
</ul>


<ul class="alternate" type="square">
	<li>Does anything appear in the Javascript console?<br/>
> A screenshot has been attached with the output in the JavaScript console.</li>
</ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>jyerabutar:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-30394?focusedId=236811&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-236811">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Got an error while trying to attach a screenshot. Below is the output in the JavaScript console.</p>
<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">GET http:<span class="code-comment">//10.39.106.250/jenkins/<span class="code-keyword">static</span>/068a7b78/images/16x16/.png 404 (Not Found)
</span>(index):1 GET http:<span class="code-comment">//10.39.106.250/jenkins/<span class="code-keyword">static</span>/068a7b78/images/16x16/.png 404 (Not Found)
</span>prototype.js:1585 POST http:<span class="code-comment">//10.39.106.250/jenkins/job/TestTimeStampPlugin/com.cloudbees.jenkins.plugins.freeplugins.Notice/instance/ajax 404 (Not Found)Ajax.Request.<span class="code-object">Class</span>.create.request @ prototype.js:1585Ajax.Request.<span class="code-object">Class</span>.create.initialize @ prototype.js:1550Object.extend.__method @ prototype.js:452klass @ prototype.js:101f @ (index):30</span>
</pre>
</div></div>
</pre>
</details>

</ul>
</details>

<p>Got an error while trying to attach a screenshot. Below is the output in the JavaScript console.</p>

<pre>
GET http:<span class="code-comment">//10.39.106.250/jenkins/<span class="code-keyword">static</span>/068a7b78/images/16x16/.png 404 (Not Found)
</span>(index):1 GET http:<span class="code-comment">//10.39.106.250/jenkins/<span class="code-keyword">static</span>/068a7b78/images/16x16/.png 404 (Not Found)
</span>prototype.js:1585 POST http:<span class="code-comment">//10.39.106.250/jenkins/job/TestTimeStampPlugin/com.cloudbees.jenkins.plugins.freeplugins.Notice/instance/ajax 404 (Not Found)Ajax.Request.<span class="code-object">Class</span>.create.request @ prototype.js:1585Ajax.Request.<span class="code-object">Class</span>.create.initialize @ prototype.js:1550Object.extend.__method @ prototype.js:452klass @ prototype.js:101f @ (index):30</span>
</pre>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/stevengbrown.png" title="stevengbrown's avatar" /> stevengbrown:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-30394?focusedId=236841&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-236841">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Thanks. Which web browser were you using?</p>
</pre>
</details>

</ul>
</details>

<p>Thanks. Which web browser were you using?</p>
